contrib/release/release-tasklist.md

```diff
@@ -198,6 +198,118 @@ and the links are working
 
 ## List of prior release notes
 
+Announcement for 3.0.0 (Nov 6, 2024)
+-----------------------------------------
```

Comment:
I think you want to move the announcement for 3.0.0 *above* the list of prior release notes? Or is this file only for us, and we keep this announcement here as a starting point for our own release process?

Reply:
This file is mostly for us and internal use to keep track of all official announcements. We can make it more prominent in the future, but then I think all of this needs to go into a different file (or files). I will keep it in place for now.